server/src/test/java/org/candlepin/dto/api/v1/ProductContentTranslatorTest.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2009 - 2017 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.AbstractTranslatorTest;
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.model.Content;
+import org.candlepin.model.ProductContent;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+
+/**
+ * Test suite for the ContentTranslator class
+ */
+public class ProductContentTranslatorTest extends
+    AbstractTranslatorTest<ProductContent, ProductContentDTO, ProductContentTranslator> {
+
+    protected ProductContentTranslator translator = new ProductContentTranslator();
+
+    @Override
+    protected void initModelTranslator(ModelTranslator modelTranslator) {
+        modelTranslator.registerTranslator(new ContentTranslator(), Content.class, ContentDTO.class);
+        modelTranslator.registerTranslator(this.translator, ProductContent.class, ProductContentDTO.class);
+    }
+
+    @Override
+    protected ProductContentTranslator initObjectTranslator() {
+        return this.translator;
+    }
+
+    @Override
+    protected ProductContent initSourceObject() {
+        Content content = new Content();
+
+        content.setUuid("test_value");
+        content.setId("test_value");
+        content.setType("test_value");
+        content.setLabel("test_value");
+        content.setName("test_value");
+        content.setVendor("test_value");
+        content.setContentUrl("test_value");
+        content.setRequiredTags("test_value");
+        content.setReleaseVersion("test_value");
+        content.setGpgUrl("test_value");
+        content.setMetadataExpiration(1234L);
+        content.setModifiedProductIds(Arrays.asList("1", "2", "3"));
+        content.setArches("test_value");
+        content.setLocked(Boolean.TRUE);
```

Comment:
Code duplication: This can be replaced by instantiating a `ContentTranslatorTest` and calling `initSourceObject()`

Reply:
Done.